countries-geojson/countries_geojson.py

```diff
@@ -0,0 +1,35 @@
+import csv, json
+from re import M
+from geojson import Feature, FeatureCollection, Point
+
+csvFilePath = "/Users/cpapalaz/Desktop/Mangrove_Countries.csv"
+jsonFilePath = "/Users/cpapalaz/Desktop/Mangrove_Countries.json"
+
+#read csv file and add to data
+features = []
+with open(csvFilePath, newline='', encoding='latin1') as csvfile:
+    print(csvfile)
```

Comment:
The `print` statement isn't required. Can you remove it? (I know I left it in my example 😅)